object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/ObjectConstructionTransfer.java

```diff
@@ -74,9 +88,54 @@ public ObjectConstructionTransfer(final CFAnalysis analysis) {
             result.getElseStore(),
             exceptionalStores);
     exceptionalStores = null;
+
+    Node receiver = node.getTarget().getReceiver();
+    addTemporaryVars(result, node);
+    if (atypefactory.biMap.inverse().containsKey(receiver)) {
```

Comment:
To me this code would be clearer if you stored the result of `atypefactory.biMap.inverse().get(receiver)` in a local variable, and then changed the condition to check if that variable is null.  Then you could avoid writing `atypefactory.biMap.inverse()` in two places